### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by sjinks**
Removed unused macros, unused files, etc

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,19 +21,20 @@ env:
...
 matrix:
   fast_finish: true
   allow_failures:
     - php: nightly
   include:
-    - env: CC="ccache gcc" CFLAGS="-g3 -O0 -fno-delete-null-pointer-checks -Wall -fvisibility=hidden -DZEPHIR_RELEASE=1"
+    - env: CC="ccache gcc"
       php: nightly
       compiler: gcc
-    - env: CC="clang" CFLAGS="-g3 -O0 -Wall -DZEPHIR_RELEASE=1"
```
Comment (sergeyklay) [inline .travis.yml:36]: @sjinks In fact the presence of `-DZEPHIR_RELEASE` flag could help to identify potential problems at development stage. isn't it?

**Comment by sergeyklay** [comment]
Fine!

File: .travis.yml (hunk 1 above)
Comment (sjinks) [reply .travis.yml:36]: No, just the opposite — `ZEPHIR_RELEASE` disables the debug code. `PHALCON_RELEASE` / `ZEPHIR_RELEASE` were used for production builds.

Comment (sergeyklay) [reply .travis.yml:36]: Sill can't understand..

We have a lot of such logic:
```c
#ifndef ZEPHIR_RELEASE
zend_error(E_WARNING, "Invalid arguments supplied for memnstr in %s on line %d", file, line);
#else
zend_error(E_WARNING, "Invalid arguments supplied for memnstr()");
#endif
```

Here `ZEPHIR_RELEASE` can help at development stage. Right? Probably we have to refactor this or use another identifier..

Comment (sjinks) [reply .travis.yml:36]: #if**n**def

That is, if `ZEPHIR_RELEASE` is NOT defined…

Comment (sergeyklay) [reply .travis.yml:36]: Oh! I'm sorry 😐

Comment (sergeyklay) [inline .travis.yml]: What about `-O0` ?

**Pull request merged**